ElementX/Sources/Screens/Timeline/View/TimelineItemViews/StickerRoomTimelineView.swift

```diff
@@ -14,15 +14,15 @@ struct StickerRoomTimelineView: View {
     
     var body: some View {
         TimelineStyler(timelineItem: timelineItem) {
-            LoadableImage(url: timelineItem.imageURL,
+            LoadableImage(url: timelineItem.imageInfo.source.url,
```

Comment:
I really can't remember if this is intentional for stickers: any ideas why we don't just
```suggestion
            LoadableImage(mediaSource: timelineItem.imageInfo.source,
```